src/webgpu/web_platform/canvas/readbackFromWebGPUCanvas.spec.ts

```diff
@@ -1,10 +1,17 @@
 export const description = `
 Tests for readback from WebGPU Canvas.
+
+TODO: implement all canvas types:
+- canvas element not in dom
+- canvas element in dom
+- offscreen canvas from transferControlToOffscreen from canvas not in dom
+- offscreen canvas from transferControlToOffscreen from canvas in dom
+- offscreen canvas from new OffscreenCanvas
```

Comment:
Probably this list should be attached to `kAllCanvasTypes` in `create_elements.ts` and the TODO should reference it